### ISSUE TITLE
TOC doesn't populate for headings greater than the first one in the content

### ISSUE THREAD

**Issue by GiridharanNarayanan** [Oktavilla/markdown-it-table-of-contents#25]
If the first heading in the page is h2 (say), if a h1 appear after it, headings h1 and below h1 are not rendered in the TOC.

Please try the below block of code in the URL: https://npm.runkit.com/markdown-it-table-of-contents 

```
var MarkdownIt = require("markdown-it");
var md = new MarkdownIt();
md.use(require("markdown-it-table-of-contents"));
var content = "## h2 Heading\n" +
" [[toc]] \n" +
"# h1 heading 1\n" +
"Some nice text\n" +
"# h1 heading 2\n" +
"Some even nicer text\n"
md.render(content)  
```

**Expected TOC:**

- h2 Heading
- h1 heading 1
- h1 heading 2

**Actual TOC:**

-  h2 Heading

 I am willing to submit a PR for this, just wanted to know if this issue is known and you would like this to be supported.

**Comment by martinlissmyr** [comment]
Hi, thanks. I appreciate all contributions! However, I'm not sure if it makes sense to support this. What kind of document starts with a h2?

**Comment by GiridharanNarayanan** [comment]
@martinlissmyr , It is not a common case. We are just trying to cover all the cases possible. In wikis like in Git hub wiki, there is a separate title field which solves the purpose of the h1. So people start with h2 post that. We have seen similar cases in our wiki too. Folks start with h2 if there is a title field for the wiki page. I agree that this is not 80% scenario, but we would wish to not exclude this scenario. It is totally fine if we can get this as an optional behavior.

**Comment by martinlissmyr** [comment]
Ok, I'll buy that... If you do a PR I'll accept it! 😄

**Comment by GiridharanNarayanan** [comment]
@martinlissmyr , I have created a PR : https://github.com/Oktavilla/markdown-it-table-of-contents/pull/27

**Comment by martinlissmyr** [comment]
Fixed by #27
